cmd/kubeadm/app/master/apiclient.go

```diff
@@ -73,11 +73,10 @@ func CreateClientAndWaitForAPI(file string) (*clientset.Clientset, error) {
 		cs, err := client.ComponentStatuses().List(v1.ListOptions{})
 		if err != nil {
 			if apierrs.IsForbidden(err) {
-				fmt.Print("\r[apiclient] Waiting for the API server to create RBAC policies")
+				fmt.Println("[apiclient] Waiting for API server authorization")
```

Comment:
@liggitt please omit this file change as we're deleting this func in #40075 and fixing it on that side

just in order to avoid rebases. Thank you